tests/providers/gcp/gcp_test.py

```diff
@@ -702,18 +699,19 @@ class GoogleCloudStorageTest(unittest.TestCase):
   @typing.no_type_check
   def testSplitGcsPath(self):
     """Tests that GCS path split is correctly done."""
-    bucket, object_uri = FAKE_GCS.SplitGcsPath(FAKE_GCS_PATH)
-    self.assertEqual(object_uri, FAKE_GCS_URI)
-    self.assertEqual(bucket, FAKE_GCS_BUCKET)
+    bucket, object_uri = FAKE_GCS.SplitGcsPath('gs://fake-bucket/fake-folder/fake-object')
```

Comment:
Probably line-too-long?

Reply:
I disabled line-too-long after starting the GoogleCloudStorageTest class, but this is already done everywhere in the gcp_test module